.circleci/config.yml

```diff
@@ -2010,14 +2066,32 @@ jobs:
             # as a temporaty solution explicitly specify ffi version.
             sudo gem install --no-document ffi:1.12.2 fpm
 
-      - run:
-          name: Bump Agent Version
-          command: |
-            # Update agent version to avoid version conflict with current scalyr repository package.
-            # This way we can also correctly assert that the upgrade has worked - if we didn't do
-            # that, from and to version would be the same
-            echo "$(cat VERSION).dev1-$(git rev-parse --short HEAD)" > VERSION
-            cat VERSION
+#      - run:
+#          name: Bump Agent Version
+#          command: |
+#            # Update agent version to avoid version conflict with current scalyr repository package.
```

Comment:
Probably a good idea to leave this comment in place so people who look at the pipeline know why that step is needed when ``agent_version`` parameter is not explicitly defined.